tlsn/examples/simple/simple_prover.rs

```diff
@@ -86,6 +101,26 @@ async fn main() {
 
     assert!(response.status() == StatusCode::OK);
 
+    // Read and print the response body
+    let body_bytes = response.into_body().collect().await.unwrap().to_bytes();
+    let body_str = String::from_utf8_lossy(&body_bytes);
+
+    println!("Response body:\n{}", body_str);
+
+    // Extract followers count
+    let followers_count = body_str
+        .lines()
+        .find(|line| line.starts_with("followers="))
+        .and_then(|line| line.split('=').nth(1))
```

Comment:
I think it may be sufficient to find the "followers=x" without splitting it because later you format it again as "followers={}"